TESTS/mbed_drivers/c_strings/main.cpp

```diff
@@ -54,6 +54,7 @@ void test_case_c_string_x_X()
     TEST_ASSERT_EQUAL_STRING("8000 C8E 3e7 64 1 0 1 1087 3e7 101B 7ff8 1869F", buffer);
 }
 
+#if !defined(__NEWLIB_NANO)
```

Comment:
It would be good to comment why we disable any tests.

Reply:
We provided the reason in the PR description. Do you prefer to add a comment?